src/highdicom/seg/sop.py

```diff
@@ -1793,10 +1612,175 @@ def _check_segment_numbers(described_segment_numbers: np.ndarray):
                 f'from 1. Found {described_segment_numbers[0]}. '
             )
 
+    @staticmethod
+    def _get_pixel_measures(
```

Comment:
Using the full attribute name may be clearer

```suggestion
    def _get_pixel_measures_sequence(
```

Reply:
resolved in [9092a00](https://github.com/ImagingDataCommons/highdicom/pull/227/commits/9092a000b3c1518b3c04c98b07d3b6682fb9a889)